examples/2.1.5.1.7-facet-json-terms.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+require_once(__DIR__.'/init.php');
+htmlHeader();
+
+// create a client instance
+$client = new Solarium\Client($adapter, $eventDispatcher, $config);
+
+// get a select query instance
+$query = $client->createSelect();
+
+// get the facetset component
+$facetSet = $query->getFacetSet();
+
+// create a json terms instance and set options
+$categoriesTerms = new Solarium\Component\Facet\JsonTerms(['key' => 'categories', 'field' => 'cat', 'limit'=>4,'numBuckets'=>true]);
```

Comment:
I that this doesn't work anymore in solarium 6 since the commit of #772.
You need to name 'key' as 'local_key'.

Luckily I recently integrated the execution of the examples in the test pipeline ;-)

Reply:
Thanks, am using a composer version of 5 in my project, will try switching to 6.